fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1689,8 +1699,8 @@ Future<RangeResult> ReadYourWritesTransaction::getRange(KeySelector begin,
 	}
 
 	Future<RangeResult> result =
-	    reverse ? RYWImpl::readWithConflictRange(this, RYWImpl::GetRangeReq<true>(begin, end, limits), snapshot)
-	            : RYWImpl::readWithConflictRange(this, RYWImpl::GetRangeReq<false>(begin, end, limits), snapshot);
+	    reverse ? RYWImpl::readWithConflictRange(this, RYWImpl::GetRangeReq(begin, end, limits, true), snapshot)
```

Comment:
`result = RYWImpl::readWithConflictRange(this, RYWImpl::GetRangeReq(begin, end, limits, reverse), snapshot)`